application/comit_node/src/http_api/rfc003/action.rs

```diff
@@ -105,6 +105,30 @@ impl FromAcceptSwapRequestHttpBody<Bitcoin, Ethereum>
     }
 }
 
+impl FromAcceptSwapRequestHttpBody<Ethereum, Bitcoin>
+    for StateMachineResponse<
+        ethereum_support::Address,
+        secp256k1_support::KeyPair,
+        bitcoin_support::Blocks,
+    >
+{
+    fn from_accept_swap_request_http_body(
+        body: AcceptSwapRequestHttpBody<Ethereum, Bitcoin>,
+        id: SwapId,
+        secret_source: &dyn SecretSource,
+    ) -> Result<Self, HttpApiProblem> {
+        match body {
+            AcceptSwapRequestHttpBody::OnlyRefund { .. } | AcceptSwapRequestHttpBody::RefundAndRedeem { .. } => Err(HttpApiProblem::with_title_and_type_from_status(400).set_detail("The refund identity for swaps where Bitcoin is the BetaLedger has to be provided on-demand, i.e. when the refund action is executed.")),
```

Comment:
🙃 I'd just say `Invalid body`.

Reply:
We already do it like this for `FromAcceptSwapRequestHttpBody<Bitcoin, Ethereum>` and I think we can afford to be very specific here since this `details` field is supposed to be ["a human-readable explanation specific to this occurrence of the problem"](https://tools.ietf.org/html/rfc7807).